components/application-broker/internal/storage/driver/memory/entity_operation.go

```diff
@@ -101,10 +110,27 @@ func (s *InstanceOperation) GetAll(iID internal.InstanceID) ([]*internal.Instanc
 	for i := range opsForInstance {
 		out = append(out, opsForInstance[i])
 	}
-
+	sort.Sort(instanceOperations(out))
```

Comment:
Add some single unit test for sorting

Reply:
I implemented two tests TestInstanceOperationFindLast and TestInstanceOperationGetAllSortOrder